lib/wcc/contentful/engine.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module WCC::Contentful
+  class Engine < ::Rails::Engine
+    isolate_namespace WCC::Contentful
+
+    config.generators do |g|
+      g.test_framework :rspec, fixture: false
+    end
+  end
+end
+
+# TODO: figure out why autoloading isn't working
```

Comment:
@jpowell couldn't figure this out.  Help?

Reply:
what aspect isn't working?

Reply:
I'm doing this right below:

```ruby
require WCC::Contentful::Engine.root.join('app/controllers/wcc/contentful/webhook_controller')
require WCC::Contentful::Engine.root.join('config/routes.rb')
require WCC::Contentful::Engine.root.join('app/jobs/wcc/contentful/delayed_sync_job')
```

Shouldn't it autoload?  But it wasn't autoloading properly in the specs